File: Basics/AddDigits.cpp
#include<iostream>

using namespace std;

int main(){
    int number;
    cout <<"Enter the digits to get its sum : ";
    cin >> number;

    while(number > 9 ){
        int answer=0, remainder;
        while(number != 0){
            remainder = number % 10;
            number /= 10;
            answer +=remainder;
        }
        number = answer;
    }

    cout << "The sum of the digits are : " << number;
}

File: Basics/AddDigits.java
public class AddDigits {
    public static void main(String[] args) {
        int num = 15;
        while ( num > 9){
            int rem,answer = 0;
            while(num !=0 ){
                rem = num % 10;
                answer =answer + rem;
                num /= 10;
            }

            num = answer;
        }

        System.out.println("The sum of the digits are : "+num);
    }
}
